sdk/storage/tests-template.yml

```diff
@@ -49,10 +49,13 @@ stages:
       CloudConfig:
         Preview:
           SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
+          ServiceConnection: azure-sdk-tests
+          SubscriptionConfigurationFilePath: eng/common/TestResources/sub-config/AzurePublicMsft.json
```

Comment:
Change this to:
          SubscriptionConfigurationFilePaths: 
            - eng/common/TestResources/sub-config/AzurePublicMsft.json